requirements.txt

```diff
@@ -22,4 +22,5 @@ jinja2>=3.0.0 # BSD
 # Those bindings must match the local GPG version and therefore this package is not installed via PyPI.
 # typing-extensions is required for TypedDict and Literal on Python < 3.8
 # as well as NotRequired on Python < 3.11
-typing-extensions>=4.0.1 ; python_version < '3.11' #PSF-2.0
\ No newline at end of file
+typing-extensions>=4.0.1 ; python_version < '3.11' #PSF-2.0
+jsonschema # MIT
```

Comment:
any constrains about the version? In SLE we have 3.2.0, as it is supporting Python 3.6. In Tumbleweed / MicroOS we have 4.17.3 that drops 3.6

Reply:
From my testing it seems that 3.2.0 is sufficient. I'll add this to the constraint.